addons/lunch/static/src/js/lunch_controller_common.js

```diff
@@ -42,7 +42,7 @@ var LunchControllerCommon = {
         // banner and renderer (this is required to get the desired
         // layout with the searchPanel to the left)
         var self = this;
-        this.$('.o_content').append($('<div>').addClass('o_lunch_content'));
+        this.$('.o_content').append($('<div>').addClass('o_lunch_content d-flex flex-column flex-grow-1 w-100 h-100'));
```

Comment:
Horizontal scrollbars should be in the table, not in the entire view (not sure if this is the line causing the issue)

![image](https://user-images.githubusercontent.com/8558351/183612033-61d65490-c384-46f6-85ea-22f8e402b756.png)